docs/snippets/svelte/button-story-click-handler.js.mdx

```diff
@@ -0,0 +1,22 @@
+```js
+// Button.stories.js
+
+import Button from './Button.svelte';
+
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Button',
+  component: Button,
+};
+
+export const Text = () =>({
+  Component:Button,
```

Comment:
We should probably document somewhere that the Component is optional is it's the same as the component in the default export.

Reply:
See if you're ok with i introduced with the latest commit.